db/migrate/20180618212608_create_cloud_volume_types.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCloudVolumeTypes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_volume_types do |t|
+      t.text :description
+      t.string :name
+      t.string :type
```

Comment:
Is this a literal STI type or some cloud-specific string?

Reply:
Intended to be STI type